templates/azure/steps/install_tiledb.yml

```diff
@@ -0,0 +1,17 @@
+parameters:
+  tiledbVersion: '2.0.8'
+
+steps:
+  - script: |
+      set -e -x
+      mkdir tiledb_core && cd tiledb_core \
```

Comment:
Since you are changing things.. shall we switch to downloading the prebuilt library to speedup CI?

Reply:
👍